solvers/Electrostatic2D3V/CMakeLists.txt

```diff
@@ -30,8 +30,9 @@ target_include_directories(
 
 target_link_libraries(
   ${ELECTROSTATIC_2D3V_LIBRARY_NAME}
-  PRIVATE ${NEKTAR++_LIBRARIES} ${NEKTAR++_TP_LIBRARIES} ${MPI_LIBRARIES}
-          ${NESO_PARTICLES_LIBRARIES} ${NESO_PARTICLES_LINK_FLAGS})
+  PRIVATE ${NESO_LIBRARY_NAME} ${NEKTAR++_LIBRARIES} ${NEKTAR++_TP_LIBRARIES}
```

Comment:
Not something introduced in this PR, but you should link against `Nektar++::nektar++` instead of the individual libraries.

Reply:
Moved the Electrostatic2D3V cmake over to `Nektar++::nektar++`.